src/legacy/core_plugins/console/public/np_ready/application/components/editor_example.tsx

```diff
@@ -32,11 +32,9 @@ export function EditorExample(props: EditorExampleProps) {
   const elemId = `help-example-${props.panel}`;
 
   useEffect(() => {
-    const el = $(`#${elemId}`);
-    el.text(exampleText.trim());
-    const editor = new SenseEditor(el);
-    editor.setReadOnly(true);
-    editor.$blockScrolling = Infinity;
+    const el = document.querySelector<HTMLElement>(`#${elemId}`)!;
+    $(el).text(exampleText.trim());
```

Comment:
I realise this isn't a "remove jQuery" PR, but you could make the following change, and then jQuery wouldn't be needed in this file. Just a thought, anyway.

```suggestion
    el.textContent = exampleText.trim();
```